testsuite/gnat2goto/tests/iss_131_vanilla/test.out

```diff
@@ -125,51 +125,53 @@ N_Identifier "vsf3_ns" (Node_Id=3880) (source,analyzed)
-[iss_131.assertion.14] line 256 assertion VnaNS (barbara)'Size = 8: SUCCESS
-[iss_131.assertion.15] line 257 assertion VnaNS (barbara .. margo)'Size = 16: SUCCESS
-[iss_131.assertion.16] line 258 assertion VnaNS (barbara .. margo)'Component_Size = 8: SUCCESS
-[iss_131.assertion.17] line 261 assertion R1'Size = 64: SUCCESS
-[iss_131.assertion.18] line 262 assertion R2'Size = 132: SUCCESS
-[iss_131.assertion.19] line 263 assertion R3'Size = 66: SUCCESS
-[iss_131.assertion.20] line 264 assertion VR1'Size = 64: SUCCESS
-[iss_131.assertion.21] line 265 assertion VR2'Size = 136: SUCCESS
-[iss_131.assertion.22] line 266 assertion VR3'Size = 72: SUCCESS
-[iss_131.assertion.23] line 267 assertion r'Size = 16: SUCCESS
-[iss_131.assertion.24] line 268 assertion Vr'Size = 16: SUCCESS
-[iss_131.assertion.25] line 273 assertion R4'Size = 64: FAILURE
-[iss_131.assertion.26] line 274 assertion R5'Size = 132: FAILURE
-[iss_131.assertion.27] line 275 assertion R6'Size = 80: FAILURE
-[iss_131.assertion.28] line 277 assertion VR4'Size = 64: SUCCESS
-[iss_131.assertion.29] line 278 assertion VR5'Size = 136: SUCCESS
-[iss_131.assertion.30] line 279 assertion VR6'Size = 72: SUCCESS
-[iss_131.assertion.31] line 284 assertion rNS'Size = 72: FAILURE
-[iss_131.assertion.32] line 285 assertion VrNS'Size = 72: FAILURE
-[iss_131.assertion.33] line 292 assertion V1'Size = 40: FAILURE
-[iss_131.assertion.34] line 294 assertion V2'Size = 39: SUCCESS
-[iss_131.assertion.35] line 295 assertion STV1'Size = 40: SUCCESS
-[iss_131.assertion.36] line 296 assertion SFV1'Size = 1: SUCCESS
-[iss_131.assertion.37] line 297 assertion VST'Size = 40: SUCCESS
-[iss_131.assertion.38] line 298 assertion VSF'Size = 8: SUCCESS
-[iss_131.assertion.39] line 306 assertion V3'Size = 64: FAILURE
-[iss_131.assertion.40] line 311 assertion V4'Size = 64: FAILURE
-[iss_131.assertion.41] line 313 assertion STV3'Size = 64: SUCCESS
-[iss_131.assertion.42] line 314 assertion SFV3'Size = 8: SUCCESS
-[iss_131.assertion.43] line 315 assertion VST3'Size = 64: SUCCESS
-[iss_131.assertion.44] line 321 assertion VSF3'Size = 32: FAILURE
-[iss_131.assertion.45] line 323 assertion VSF3'Size = 8: SUCCESS
-[iss_131.assertion.46] line 328 assertion STV3_No_Size_Rep'Size = 64: FAILURE
-[iss_131.assertion.47] line 329 assertion SFV3_No_Size_Rep'Size = 8: FAILURE
-[iss_131.assertion.48] line 330 assertion VST3_NS'Size = 64: FAILURE
-[iss_131.assertion.49] line 331 assertion VSF3_NS'Size = 32: FAILURE
-[iss_131.assertion.50] line 334 assertion A_T'Size = 8: SUCCESS
-[iss_131.assertion.51] line 335 assertion VA'Size = 8: SUCCESS
-[iss_131.assertion.52] line 336 assertion VA (1)'Size = 2: SUCCESS
-[iss_131.assertion.53] line 337 assertion VA (0 .. 2)'Size = 8: SUCCESS
-[iss_131.assertion.54] line 339 assertion Vna'Component_size = 1: SUCCESS
-[iss_131.assertion.55] line 340 assertion neigh_array'component_size = 1: SUCCESS
+[iss_131.assertion.9] line 252 assertion Vna (barbara)'size = 1: FAILURE
```

Comment:
This seems to have changed status.  I am not sure if there are others.  Is this intended?

Reply:
Yes, there are some changes because we use the model size instead of the front-end derived size.

Reply:
OK, as long as you are happy with these that is fine.